01-intro.Rmd

```diff
@@ -71,17 +71,17 @@ The following outline several key questions:
 The DMS Policy applies to all research that generates [scientific data](#scientific-data), including:
 
 - Research Projects
-- Some Career Development Awards (Ks)
+- Some Career Development Awards (K)
 - Small Business SBIR/STTR 
 - Research Centers
 
 The DMS Policy does not apply to research and other activities that do not generate scientific data, including:
 
 - Training (T)
-- Fellowships (Fs)
+- Fellowships (F)
```

Comment:
This is what I found surprising during their presentation... I think they just mean the funding to support training... because don't Fs and Ts often result in data generation??

Reply:
maybe should add a question about this to our list?

Reply:
so maybe we could add some clarification about this - that the research the trainees do that does generate data funded some other way still needs to be shared?

Reply:
I can add this to the list of questions!